public/data/sumitmukharjeeeeee.json

```diff
@@ -1,15 +1,17 @@
 {
-    "name": "Sumit Mukharjee",
-    "bio": "Learning and Exploring",
-    "avatar": "https://github.com/sumitmukharjeeeeee.png",
-    "links": [{
-        "name": "Follow me on GitHub",
-        "url": "http://github.com/sumitmukharjeeeeee",
-        "icon": "github"
-    }, {
-        "name": "Follow me on Twitter",
-        "url": "http://twitter.com/SumitMukharjii",
-        "icon": "twitter"
+  "name": "Sumit Mukharjee",
+  "bio": "Learner",
+  "avatar": "https://github.com/sumitmukharjeeeeee.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sumitmukharjeeeeee",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SumitMukharjii",
+      "icon": "twitter"
     }
- ]
+  ]
```

Comment:
@sumitmukharjeeeeee sorry, now it is correctly formatted, that can be moved back a space. Thank you.

Reply:
Is it correct now?

Reply:
it is all good, but the square bracket needs moving back. thank you.

Reply:
For me, this is already fine. It is correct as-is.

Reply:
RE: We can click the resolve button after the issues are fixed 🙂